tests/conftest.py

```diff
@@ -151,6 +151,10 @@ def getAllDependencies(filePath, knownDependencies):
         matches = findall("import ['\"](.*?)['\"]", fileContents)
         for match in matches:
             dependencyPath = path.join(BASE_PATH, '..', 'source/contracts', match)
+            if not path.isfile(dependencyPath):
```

Comment:
Instead of checking if the file exists, we could instead just check to see if the filename is prefixed with `TEST`.  This will avoid having to go to disk to probe for the file when we can quickly know the path won't be found.
```
dependencyPath = path.join(BASE_PATH, '..', 'source/contracts', match) if path.startsWith('TEST') else path.join(BASE_PATH, 'solidity_test_helpers', match).replace("TEST/", "")
```
(or however Python ternarys works)

Reply:
updated to use "TEST" in dependency python's "contains"